distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -1139,10 +1220,24 @@ public void testCacheIsUpdatedIfRemoteProductChanges() throws Exception {
         when().get(restUrl).then().log().all().assertThat().contentType("text/plain")
                 .body(is(STUB_SERVER_TEST_STRING));
         // @formatter:on
-        cswServer.verifyHttp()
-                .times(2,
-                        Condition.uri("/services/csw"),
-                        Condition.parameter("request", "GetRecordById"));
+        cswServer.verifyHttp().times(2, Condition.uri("/services/csw"), Condition.parameter("request", "GetRecordById"));
+
+        expect("Waiting for notifications. Received " + cometDClient.getMessages(NOTIFICATIONS_CHANNEL).size() + " of 2").within(10, SECONDS).until(() -> cometDClient.getMessages(NOTIFICATIONS_CHANNEL).size() == 2);
```

Comment:
Since the notifications and activities are the same for both downloads, and since we already have a test that tests a single download, do we need to have all these validations here?